### PR TITLE
Update preferences to the interests page and added some styles

### PR DESCRIPTION
**Pull request by fernanda1411**
I'll work on some enhancements on a following PR.

image

image

image

### CODE REVIEW

File: client/src/components/interests.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Box, Flex, Heading, extendTheme } from '@chakra-ui/react';
 import React from 'react';
 import { Header } from './header';
-
+import Preferences from './preferences/Preferences.jsx';
```
Comment (awhlam) [inline client/src/components/interests.js:4]: Could `interests.js` be replaced with `Preferences.jsx`?

Comment (fernanda1411) [reply client/src/components/interests.js:4]: Preferences is a component which is rendered within the route interests.

Comment (awhlam) [reply client/src/components/interests.js:4]: Gotcha. I assume a future PR will render it inside the interests page since you're importing it here?

Comment (fernanda1411) [reply client/src/components/interests.js:4]: It's there. I've already sent and update. Somehow I've deleted it before. Thanks for catching it.

Comment (awhlam) [reply client/src/components/interests.js:4]: No problem, looks good now :)

**Review by awhlam — APPROVED** [review]
I pulled this down and tested it locally. Looks good, will just need to resolve the merge conflicts.

image

**Pull request merged**